recipes/poco/all/conanfile.py

```diff
@@ -108,6 +108,9 @@ def configure(self):
             raise ConanInvalidConfiguration("Apache connector not supported: https://github.com/pocoproject/poco/issues/1764")
         if self.options.enable_data_mysql:
             raise ConanInvalidConfiguration("MySQL not supported yet, open an issue here please: %s" % self.url)
+        if self.settings.compiler == "Visual Studio":
+            if self.options.shared and "MT" in str(self.compiler.runtime):
```

Comment:
@uilianries I just tried without `str` and it is needed to check against `MT/MTd`.

Reply:
@madebr thanks for checking! I'll remember it!